### PR TITLE
Flwagner/va on aks

### PR DESCRIPTION
**Pull request by h2floh**
Terraform Dev Infrastructure Setup (partly)

### CODE REVIEW

File: tools/terraform/dev/cosmosdb.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Not possible to create a collection - only up to database
```
Comment (angie4u) [inline tools/terraform/dev/cosmosdb.tf]: How about removing this file if we don't use this file. 😁

**Review by angie4u — COMMENTED** [review]
How about delete unused file or do you want to leave those file as reference?

File: tools/terraform/dev/cosmosdb.tf (hunk 1 above)
Comment (h2floh) [reply tools/terraform/dev/cosmosdb.tf]: Good point, removed 2 empty files. QnAMaker and ComsosDB can not be created like we expected.

**Review by angie4u — APPROVED** [review]
It looks good! Thank you for the commit

**Pull request merged**